osu.Game/Overlays/BeatmapSet/Scores/ScoresContainer.cs

```diff
@@ -62,7 +62,7 @@ public BeatmapInfo Beatmap
                 loading = true;
 
                 getScoresRequest = new GetScoresRequest(beatmap, beatmap.Ruleset);
-                getScoresRequest.Success += r => Scores = r.Scores;
+                getScoresRequest.Success += r => Schedule(() => Scores = r.Scores);
```

Comment:
Purposefully not disposed?

Reply:
What do you mean by disposed? cancelled?

Reply:
Yeah

Reply:
It gets cancelled when new scores are set on it. The `ScoresContainer` has a lifetime of forever.

we could cancel in dispose as a safety measure, won't really affect anything.